pydis_site/templates/content/base.html

```diff
@@ -33,7 +34,17 @@
     <section class="section">
         <div class="container">
             <div class="content">
-                <h1 class="title">{{ page_title }}</h1>
+                <div class="is-flex" style="justify-content: space-between;">
+                    <h1 class="title is-flex">{{ page_title }}</h1>
+                    {# TODO: Bikeshed the button design #}
+                    <button class="button is-flex is-pulled-right is-primary is-light">
+                        {# TODO: Make it load the Github page instead of reloading the page #}
+                        <a href="{{ request.build_absolute_uri|page_github }}">
```

Comment:
If you use [`request.path`](https://docs.djangoproject.com/en/3.2/ref/request-response/#django.http.HttpRequest.path) instead of `request.build_absolute_uri` you can avoid the URL hassle and just strip `/pages/` from the path.